src/Context/CategoryContext.jsx

```diff
@@ -0,0 +1,12 @@
+import { createContext ,useContext,useReducer} from "react";
+import { reducerFunc } from "../Reducer/CategoryReducer";
+
+const categoryContext=createContext();
+
+const CategoryProvider=({children})=>{
+    const[state,dispatch]=useReducer(reducerFunc,{categoryData:[]})
+return(<categoryContext.Provider value={{state,dispatch}}>{children}</categoryContext.Provider>)
+}
+const useCategory=()=>useContext(categoryContext);
+
+export {useCategory,CategoryProvider}
```

Comment:
Export on while declaration instead of exporting useCategory and CategoryProvider in the end.

`export const useCategory=()=>useContext(categoryContext);
`